src/components/notifications/NotificationsMenu.js

```diff
@@ -223,29 +260,12 @@ function NotificationsMenu(props) {
                 ))}
             {!isMobile && [
                 <Divider light key="divider" />,
+                <NotificationsListingLink
+                    key={ids.VIEW_ALL_NOTIFICATIONS}
+                    handleClose={handleClose}
+                />,
```

Comment:
We need Link for IconButtons in mobile https://github.com/cyverse-de/sonora/blob/4dac918843928b3191c4698116d712232635c8d1/src/components/notifications/NotificationsMenu.js#L160